django/db/migrations/operations/utils.py

```diff
@@ -14,6 +14,31 @@ def is_referenced_by_foreign_key(state, model_name_lower, field, field_name):
     return False
 
 
+def describe_operation(operation, backwards):
```

Comment:
I'm thinking this would be better organized as a method of the migrate command.

Reply:
Sure. I'll move it there.